htdocs/web_portal/views/start_page.php

```diff
@@ -36,8 +36,8 @@
     <!-- map Block -->
     <?php if($params['showMap']): ?>
       <!-- Use a web based leaflet.js to avoid distributing leaflet.js. -->
-      <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.4.0/leaflet.css" />
```

Comment:
The types aren't needed?

Reply:
I guess not, the SRI-enabled version I copied from the CDN rather than a random web tutorial